content/general/concepts/function/function.md

```diff
@@ -0,0 +1,30 @@
+---
+Title: 'Function'
+Description: 'The function data type.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Data Types'
+  - 'Types'
+CatalogContent:
+  - 'paths/code-foundations'
+  - 'paths/computer-science'
+---
+
+In programming, functions hold a distinct place among the [data types](https://www.codecademy.com/resources/docs/general/data-types). Functions are blocks of code that yield results or perform a specific task. In some programming languages, they are processed as objects of data. This is because in some languages they are derived from objects and its closest classification is with composite data types.
```

Comment:
```suggestion
Functions hold a distinct place among the [data types](https://www.codecademy.com/resources/docs/general/data-types). They are blocks of code that yield results or perform a specific task. In some programming languages, they are processed as objects of data. This is because in some languages they are derived from objects and its closest classification is with composite data types.
```

I'm wondering how functions "are processed as objects of data" results in categorizing them as a composite data type. Could we either expand on this idea or choose another transitional phrase?

Reply:
@Dusch4593 Wondering about your opinion on these last two sentences 🧠